gulpfile.babel.js

```diff
@@ -112,7 +112,7 @@ gulp.task('build:search', ['build:status'], () => {
   fs.writeFileSync(searchFilePath, JSON.stringify(searchFeatures));
 });
 
-gulp.task('build:html', ['build:index', 'build:features', 'build:css'], () => {
+gulp.task('build:html', ['build:index', 'build:features', 'build:css'], () =>
```

Comment:
Yeah, I think the function should return the result of `gulp.src`, otherwise the task will not wait for `gulp.src` to finish.